fitbenchmarking/jacobian/tests/test_jacobian.py

```diff
@@ -94,6 +95,20 @@ def test_scipy_cs_eval(self):
         eval_result = jac.eval(params=self.params)
         self.assertTrue(np.isclose(self.actual, eval_result).all())
 
+    def test_numdifftools_eval(self):
+        """
+        Test whether numdifftools evaluation is correct
+        """
+        for method in ['central',
+                       'forward',
+                       'backward',
+                       'complex',
+                       'multicomplex']:
+            jac = numdifftools(self.cost_func)
+            jac.method = method
```

Comment:
I was looking at why this wasn't in the constructor for Jacobian, but I think that's something to look into separate to this PR.